Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealityProfileCloneWindow.cs

```diff
@@ -404,10 +404,12 @@ private static Object EnsureTargetFolder(Object targetFolder)
             {
                 // AssetDatabase.CreateFolder must be called to create each child of the asset folder
                 // path individually. 
-                // Calling AssetDatabase.CreateFolder("Assets", "MixedRealityToolkit.Generated/CustomProfiles")
-                // generates a folder that looks like "Assets/MixedRealityToolkit.Generated_CustomProfiles".
-                AssetDatabase.CreateFolder("Assets", "MixedRealityToolkit.Generated");
+                
+                // Create the MixedRealityToolkit.Generated folder at the root of the Assets
+                MixedRealityToolkitFiles.TryToCreateGeneratedFolder("Assets/");
+
                 AssetDatabase.CreateFolder("Assets/MixedRealityToolkit.Generated", "CustomProfiles");
```

Comment:
This also needs to be fixed. We should not be assuming where the Generated folder should be. It should be retrieve via the MRTKFiles.cs class

Reply:
The file location is retrieved via MapModulePath